Dockerfile

```diff
@@ -0,0 +1,28 @@
+# CodeClimate specification: https://github.com/codeclimate/spec/blob/master/SPEC.md
+#
+# Build and run via:
+#   docker build -t codeclimate/codeclimate-reek . && docker run codeclimate/codeclimate-reek
+
+FROM codeclimate/alpine-ruby:b38
```

Comment:
Ah, sorry, my thinking was clouded by my clogged sinuses – if `alpine:edge` (which `codeclimate/alpine-ruby:b38` uses) gets updated it’s enough to rebuild all of the descendant images, no need to bump their versions, right? If so then this is ok (although I wonder whether it shouldn’t be `codeclimate/alpine-ruby:latest` anyway).